[Trivia] 2020 Trivia List

OofChair opened the pull request:
### Type

- [ ] Bugfix
- [ ] Enhancement
- [X] New feature

### Description of the changes

There's a 2015, and a 2016 trivia list, so why not make a 2020 one
Currently there are 20 questions
Sources: [Source 1](https://www.nydailynews.com/news/national/ny-2020-timeline-events-20200705-lvu3hluqdnh3nedmvkpteceqcm-story.html), [Source 2](https://nypost.com/list/major-2020-events/), and [Source 3](https://www.onthisday.com/events/date/2020?p=5)

OofChair committed:
2020 trivia list :0

There's a 2015, and a 2016 trivia list, so why not make a 2020 one 
Might add more things, but I think this is good for right now
Currently there are 20 questions
Sources: https://www.nydailynews.com/news/national/ny-2020-timeline-events-20200705-lvu3hluqdnh3nedmvkpteceqcm-story.html, https://nypost.com/list/major-2020-events/, https://www.onthisday.com/events/date/2020?p=5

Jackenmen commented on redbot/cogs/trivia/data/lists/2020.yaml:
```diff
@@ -0,0 +1,97 @@
+- January 8
+- Jan 8th
+- January 8th
+When did COVID-19 first appear?:
```
I'm guessing you meant the first appearance outside China?

OofChair replied:
First appearance inside China

Jackenmen replied:
Then I don't think that's true, [according to WHO](https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200423-sitrep-94-covid-19.pdf) the first human cases were reported in early December 2019.

OofChair replied:
Ah yes, I am seeing that one of the earliest cases was in December 2019.

aikaterna commented on redbot/cogs/trivia/data/lists/2020.yaml:
```diff
@@ -0,0 +1,97 @@
+- July 15
+- Jul 15th
+- July 15th
+When were the murder first spotted in the US?:
```
What even is this question?

OofChair replied:
meant murder hornets, updated

aikaterna commented:
IMO, this needs more questions, the list is too short... it is too COVID-focused and also politics focused. There are more things that happened in 2020 besides these topics.

OofChair committed:
murder HORNETS *oops

NeuroAssassin commented:
In addition to what aika said, I was wondering if we could have more questions that did not ask about dates?  If you look at the other 2015 and 2016 lists, they are largely diverse on answer choices.

OofChair committed:
updated first covid appearance

maybe remove this because it was in 2019?

OofChair commented:
> IMO, this needs more questions, the list is too short... it is too COVID-focused and also politics focused. There are more things that happened in 2020 besides these topics.

I thought it might be a little short, wasnt sure
I do agree, it is a little bit COVID and politics based, was looking more at US things than whole world things so that's probably why. 

> In addition to what aika said, I was wondering if we could have more questions that did not ask about dates? If you look at the other 2015 and 2016 lists, they are largely diverse on answer choices.

I did see that, I was thinking about that. 
I'm going to implement these changes over the next few days, should I close this one and make a new one when I'm ready?

Kreusada commented:
Agreed, dates aren't the best thing to quiz about. All these major 2020 events occurred, but people will be more aware of the context of these events rather than their time of occurrence. Also, I'd add a lot more questions, focusing on a wider range of events that occurred in 2020, instead of focusing on prominence. Best of luck

Drapersniper commented:
A much as I am in favour of a new trivia, I do believe this might be a poor choice should it be approved.

It's too US, death and Political centric for my own taste, and as a non US citizen there's a lot of negative/polarising information in there which I wouldn't want distributes in my own servers.

ltzmax commented:
I find this a bit scary, on how 2020 have been like is actually nothing that I want on my bot with this. I would rather do an 2019 over 2020. This year hasn't been the best. 

But yea more quest and little less dates would be a bit better maybe.

OofChair commented:
Thanks everyone for your feedback. So here's some things I should do, is this correct?

- Make it less US
- Less death and political things
- Less/no questions with dates

Like @maxbooiii said, I'm thinking about making a 2019 trivia list instead, that would be better.
If this is correct, I will begin reworking on this 👍🏻

Drapersniper commented:
> Thanks everyone for your feedback. So here's some things I should do, is this correct?
> 
> * Make it less US
> * Less death and political things
> * Less/no questions with dates
> 
> Like @maxbooiii said, I'm thinking about making a 2019 trivia list instead, that would be better.
> If this is correct, I will begin reworking on this 👍🏻

Really up to you, I believe a 2020 is doable with question like:
"In 2020 Steam broke its own record of concurrent online players, how many players were online ?"  And other trivia questions that are not country specific or pandemic specific.

bobloy commented:
My contribution to this list, some things that aren't all politics:
https://mashable.com/article/what-happened-2020/

OofChair commented:
> My contribution to this list, some things that aren't all politics:
> https://mashable.com/article/what-happened-2020/

This will help, thank you :)

OofChair committed:
first commit

changed up the questions, less dates, death, and politics, will add more questions later/tomorrow

ltzmax commented on redbot/cogs/trivia/data/lists/2020.yaml:
```diff
@@ -0,0 +1,48 @@
+- june
+- June
+What streaming service had their subscriber count doubled during the first 3 months of 2020?:
+- Netflix
```
You've added `Netflix` 2 times, one here and below this one, maybe remove one of them? since they does not look different :P

OofChair replied:
Thanks MAX, but I think I'm gonna close this because I have no time to work on this currently because of life things, and I might reopen later.
